tests/plugins/test_loader.py

```diff
@@ -51,8 +49,7 @@ def get_supported_configs(self) -> list[VariantFeatureConfigType]:
 class ExceptionPluginBase(PluginType, ABC):
     namespace = "exception_test"
 
-    @abstractproperty
```

Comment:
Any reason to un-abstract it? Not a big deal but wondering if the type checker had an issue. Also, since there is nothing abstract left here, we should remove `ABC` inherit.